report/src/sections/01-introduction.tex

```diff
@@ -51,21 +51,49 @@ \subsection{Non-terminating Analysis}\label{subsec:non-terminating-analysis}
 \subsection{Contributions}\label{subsec:contributions}
 \todo[inline]{Casper says: These section (\ref{subsec:contributions}, \ref{subsec:related-work}, \ref{subsec:article-overview}) read like notes, I assume they are.}
 
-Use Halder concretely for abstract interpretation
+%Use Halder concretely for abstract interpretation:
+This paper builds on the work of abstract interpretation of database queries by Cortesi and Halder.
+Concretely, we use their work to analyze a database schema concerning a program using an alteration of the syntax they describe.
```

Comment:
"Alternation" refers to the process of repeatedly changing from one state, condition, action, or place to another and back again.